staging/src/k8s.io/client-go/util/cert/cert.go

```diff
@@ -53,6 +54,27 @@ type AltNames struct {
 	IPs      []net.IP
 }
 
+// RemoveDuplicateAltNames removes duplicate items in altNames.
+func RemoveDuplicateAltNames(altNames *AltNames) {
```

Comment:
i think i saw a unit test for the old function? we should still have one (e.g. called TestRemoveDuplicateAltNames)

Reply:
Sorry for missing that.